subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -115,6 +120,17 @@ uint8_t ll_big_create(uint8_t big_handle, uint8_t adv_handle, uint8_t num_bis,
 	if (num_bis != 1) {
 		return BT_HCI_ERR_MEM_CAPACITY_EXCEEDED;
 	}
+
+	/* Add ACAD to AUX_SYNC_IND */
+	err = ull_adv_sync_acad_enable(lll_adv_sync,
+				       (sizeof(struct pdu_big_info) + 2),
```

Comment:
What's the "magic" `+ 2` here?

Reply:
2 bytes for the Advertising Data Format (length, and Advertising Data Type from assigned numbers in CSS v9 overhead). If there is something followed in Host, please let me know I will make changes to be consistent with Host conventions)

Reply:
In the host we have the struct

```
struct bt_data {
	uint8_t type;
	uint8_t data_len;
	const uint8_t *data;
};
```
So we don't really have any specific way of handling this scenario. I do see that in the host we do have a couple of length checks that check for `< 2`.

I'd suggest just adding a comment here stating what the `+ 2` is (or creating a define something like `#define BT_ADV_DATA_HDR_SIZE 2`) - Such a define would be useful for the host as well - Maybe next to `BT_HCI_EVT_HDR_SIZE` in hci.h?

Reply:
The host format is not length encoded structure.

BTW, the `struct pdu_big_info` needs to be variable size, encryption or no encryption is based on size of pdu_big_info (i.e. the presence of the optional giv and gskd fields. There is more work to do. I will add comments after I make changes to consider encryption.